discovery/service.go

```diff
@@ -526,12 +627,219 @@ func (d *Discovery) processNetworkAnnouncement(aMsg *networkMsg) bool {
 		}
 
 		if err := d.cfg.Router.UpdateEdge(update); err != nil {
-			log.Errorf("unable to update edge: %v", err)
-			return false
+			e := errors.Errorf("unable to update edge: %v", err)
+			if routing.IsError(err,
+				routing.ErrOutdated,
+				routing.ErrIgnored) {
+				log.Info(e)
+			} else {
+				log.Error(e)
+			}
+
+			nMsg.err <- e
+			return nil
 		}
-	}
 
-	return true
+		if !nMsg.isRemote {
+			remotePubKey := oppositePubKey
+			d.cfg.SendToPeer(remotePubKey, msg)
```

Comment:
A `ChannelUpdateAnnouncement` doesn't need to be sent to the remote side before a channel is considered to be open. Instead, if accepted, it should simply be added to the next batch and re-broadcast along with the rest of the newly accepted announcements.